lib/remote/apilistener.cpp

```diff
@@ -429,12 +429,19 @@ void ApiListener::ListenerCoroutineProc(boost::asio::yield_context yc, const Sha
 	namespace asio = boost::asio;
 
 	auto& io (IoEngine::Get().GetIoContext());
+	time_t lastModified = 0;
 
 	for (;;) {
 		try {
-			auto sslConn (Shared<AsioTlsStream>::Make(io, *sslContext));
+			asio::ip::tcp::socket socket (io);
+
+			server->async_accept(socket.lowest_layer(), yc);
 
-			server->async_accept(sslConn->lowest_layer(), yc);
+			if (lastModified != Utility::GetFileCreationTime(GetCrlPath()))
+				UpdateSSLContext();
```

Comment:
UpdateSSLContext() disconnects all already established connections. Do we want this?